src/components/BoardOptionsMenu/DeleteModal/DeleteModal.tsx

```diff
@@ -7,21 +7,21 @@ const DeleteModal = ({ closeDeleteModal, handleDeleteBoard }: DeleteModalProps)
       className="z-50 min-h-full min-w-full flex fixed top-0 left-0 justify-center items-center bg-black bg-opacity-30"
     >
       <div className="relative px-4 w-full max-w-sm h-full md:h-auto mb-36">
-        <div className="relative bg-trello-gray-100 rounded-lg shadow dark:bg-gray-700">
```

Comment:
places that had these `dark:...` classes have been removed as they should be here